socorro/external/es/crashstorage.py

```diff
@@ -222,9 +249,9 @@ def get_keys_for_fields(self):
         return {
             "%s.%s" % (field["namespace"], field["in_database_name"])
             for field in FIELDS.values()
+            if field.get("storage_mapping")
```

Comment:
The mappings are built from the storage mapping for fields. If the field has no storage_mapping set, then nothing gets added to the mapping and if the field is indexed, Elasticsearch infers what to do with it. Which is totally not the situation we want to be in.

This PR changes how it all works. From now on, if it's got no storage_mapping specified, then it doesn't get indexed.